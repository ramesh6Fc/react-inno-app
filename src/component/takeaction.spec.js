import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Takeaction from './takeaction';


describe('The Takeaction Component', () => {
    it ('Takeaction component page renderer as expected',() => {
        const tree = renderer
                        .create(<Takeaction />)
                        .toJSON();
                        console.log(tree.children, "Takeaction component page elements");

                        expect(tree).toMatchSnapshot ();
    })

})