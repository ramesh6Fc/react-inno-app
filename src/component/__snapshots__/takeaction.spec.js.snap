// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Takeaction Component Takeaction component page renderer as expected 1`] = `
<div
  className="container-fluid"
>
  <ul
    className="d-flex justify-content-around align-middle"
    style={
      Object {
        "list-style": "none",
      }
    }
  >
    <li
      className="email social-link  social-net"
    >
      <a
        href="/#"
      >
        <i
          aria-hidden="true"
          className="fa fa-envelope"
        />
        Â STAY AHEAD
      </a>
    </li>
    <li
      className="social-link  social-net"
    >
      <a
        href="/#"
      >
        <i
          aria-hidden="true"
          className="fa fa-facebook"
        />
         
      </a>
      <a
        href="/#"
      >
        <i
          aria-hidden="true"
          className="fa fa-twitter"
        />
         
      </a>
      <a
        href="/#"
      >
        <i
          aria-hidden="true"
          className="fa fa-linkedin"
        />
         
      </a>
    </li>
    <li
      className="takeaction social-link"
    >
      <a
        href="/#"
      >
        <i
          aria-hidden="true"
          className="fa fa-search"
        />
         
      </a>
      <a
        className="btn bg-green text-black"
        href="/#"
        style={
          Object {
            "fontSize": "13px",
          }
        }
      >
        TAKE ACTION
      </a>
    </li>
  </ul>
</div>
`;
